src/dsymbol.d

```diff
@@ -1207,6 +1207,9 @@ private:
     Dsymbols* importedScopes;
     PROTKIND* prots;            // array of PROTKIND, one for each import
 
+    import std.bitmanip : BitArray;
+    BitArray accessiblePackages;// whitelist of accessible (imported) packages
```

Comment:
As if the layout of BitArray would ever need to change.
We did clarify during the ddmd switch that D w/o phobos doesn't make sense, b/c it won't bring the productivity improvements we're targeting for.
At the same time we shouldn't use edgy phobos features to avoid bootstrap hassles, but compatibility is guaranteed by testing against all the old compiler versions.